stdlib/public/SDK/Intents/INSetProfileInCarIntent.swift

```diff
@@ -128,6 +128,11 @@ extension INSetProfileInCarIntent {
         return __defaultProfile?.boolValue
     }
 
+    @available(iOS, deprecated: 11.0, message: "Please use isDefaultProfile instead")
```

Comment:
Should these deprecations really be based on deployment target and not Swift version?